lib/runtime-c-api/Cargo.toml

```diff
@@ -15,16 +15,21 @@ crate-type = ["cdylib", "rlib", "staticlib"]
 libc = "0.2.60"
 
 [dependencies.wasmer-runtime]
+default-features = false
 path = "../runtime"
 version = "0.6.0"
 
 [dependencies.wasmer-runtime-core]
+default-features = false
 path = "../runtime-core"
 version = "0.6.0"
 
 [features]
+default = ["cranelift-backend"]
 debug = ["wasmer-runtime/debug"]
-llvm = ["wasmer-runtime/llvm"]
+cranelift-backend = ["wasmer-runtime/cranelift", "wasmer-runtime/default-backend-cranelift"]
+llvm-backend = ["wasmer-runtime/llvm"]
+singlepass-backend = ["wasmer-runtime/default-backend-singlepass", "wasmer-runtime-core/backend-singlepass"]
```

Comment:
I think this is:

```diff
- wasmer-runtime-core/backend-singlepass
+ wasmer-runtime/backend-singlepass
```

Thoughts?

Reply:
checking shortly

Reply:
@Hywan I seem it should stay `wasmer-runtime/default-backend-singlepass`

https://github.com/wasmerio/wasmer/blob/master/lib/runtime/Cargo.toml#L41

Reply:
I'm talking about the second item of the list. You are using  `wasmer-runtime-core/…`, I think it is `wasmer-runtime/…`.